Server/VikingVault.Services/TransferFundsService.cs

```diff
@@ -33,14 +33,12 @@ public void TransferFunds(TransferFundsModel transferData)
         {
             int idSender = transferData.Sender.Id;
             int? idReciever = _userCardService.FindUserIdByCardNumber(transferData.CardNumberReciever);
-            if(idReciever != null) {
-                if (UsersHaveCardsAttached(idSender, idReciever) && AreDifferentUsers(idSender, (int)idReciever))
-                {
-                    _bankAccountService.RetractMoneyFromUser(_userService.GetById(idSender), transferData.Currency, transferData.AmountSent);
-                    _bankAccountService.AddMoneyToUser(_userService.GetById((int)idReciever), transferData.Currency, transferData.AmountSent);
-                    _transactionService.AddTransactionsForTransferFunds(transferData);
-                    AddReceivedNotification(transferData, idReciever);
-                }
+            if (UsersHaveCardsAttached(idSender, idReciever) && AreDifferentUsers(idSender, (int)idReciever))
```

Comment:
You potentially try to do `(int)null` since `idReciever` is nullable

Reply:
Done. Threw exeption above if